storage/client.go

```diff
@@ -356,6 +400,66 @@ func (c Client) exec(verb, url string, headers map[string]string, body io.Reader
 		body:       resp.Body}, nil
 }
 
+func (c Client) execInternalJSON(verb, url string, headers map[string]string, body io.Reader) (*odataResponse, error) {
+	req, err := http.NewRequest(verb, url, body)
+	for k, v := range headers {
+		req.Header.Add(k, v)
+	}
+
+	httpClient := http.Client{}
```

Comment:
it still looks like this part is not addressed. can you just `git fetch origin && git rebase origin/master` and `git push -f`